test/new-e2e/pkg/utils/e2e/e2e.go

```diff
@@ -392,6 +392,9 @@ type Suite[Env any] struct {
 	env *Env
 
 	isUpdateEnvCalledInThisTest bool
+
+	// DevMode indicates if the suite is running in development mode.
+	DevMode bool
```

Comment:
We will probably not add this here, because the `params params.Params` field of this struct already include a `DevMode` attribute that you can probably reuse in your use case.
We currently use it to skip stack deletion at the end of the test when DevMode is activated. The DevMode can be enabled with `params.WithDevMode` as you do in your tests.
To access this param from your suite I'd suggest modifying the `IsDevMode()` method you added to return `suite.params.DevMode`

Reply:
If you really need to define a `DevMode` that is different from the one we currently use, you should embed it in your custom Suite for logs tests